src/Products/SiteErrorLog/SiteErrorLog.py

```diff
@@ -249,7 +235,7 @@ def _do_copy_to_zlog(self, now, strtype, entry_id, url, tb_text):
                             now - _rate_restrict_burst * _rate_restrict_period)
             next_when += _rate_restrict_period
             _rate_restrict_pool[strtype] = next_when
-            LOG.error('%s %s\n%s' % (entry_id, url, tb_text.rstrip()))
+            LOG.error(f'{entry_id} {url}\n{tb_text.rstrip()}')
```

Comment:
For logging statements, I think it is still better to use the percentage syntax: it avoids needless formatting when logging for this level is switched off.
But if this is done with an automatic code formatter, like `pyupgrade` I wouldn't bother checking this.

I thought pyupgrade was smart about this though.
Ah, I see: in the original code it is already first parsed as string, and then passed to `LOG.error` without extra arguments. So the original code was not optimal.
Could be fixed, but I don't mind.

Reply:
IMHO it's better to be consistent, and in this case, who would switch off level `ERROR`? I would agree if this was something like `DEBUG` or `TRACE`, though.